python/konduit/inference.py

```diff
@@ -148,6 +148,9 @@ def python_step_func(
     output_column_names=None,
     output_types=None,
 ):
+    if not bool(input_name and input_name.strip()): 
```

Comment:
Testing also for whitespace here, if that's too strict happy to revert to `if not input_name:`